### ISSUE TITLE
Named Alternates

### ISSUE THREAD

**Issue by edent** [Automattic/wordpress-activitypub#941]
As per https://core.trac.wordpress.org/ticket/59006

Alternates should be named so that screen readers, text browsers, and others can tell what they are.

These lines:

https://github.com/Automattic/wordpress-activitypub/blob/12ab7768d431d5a38d8f9d4c223824580074451d/includes/class-activitypub.php#L165-L175

Causes a "blank" alternate on my site.
![Image](https://github.com/user-attachments/assets/de299453-c7e0-4655-9013-9c69f1ccbc0b)

I suggest adding `title="ActivityPub."`
